src/hotspot/share/code/nmethod.hpp

```diff
@@ -472,7 +472,7 @@ class nmethod : public CodeBlob {
 
 public:
   // If you change anything in this enum please patch
-  // vmStructs_jvmci.cpp and HotSpotNmethod.java accordingly.
+  // vmStructs_jvmci.cpp accordingly.
   enum class ChangeReason : u1 {
```

Comment:
I would also suggest this enum be changed to `InvalidationReason` but that can be considered as a potential follow up.